apps/user-service/src/main/resources/application-dev.yaml

```
@@ -1,3 +1,7 @@
 spring:
   jpa:
     show-sql: true
+
+backend:
+  host:
+    url: http://localhost:8084
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)